packages/twenty-server/src/engine/api/graphql/workspace-query-runner/factories/query-runner-args.factory.ts

```diff
@@ -75,18 +75,20 @@ export class QueryRunnerArgsFactory {
       case ResolverArgsType.FindDuplicates:
         return {
           ...args,
-          id: await this.overrideValueByFieldMetadata(
-            'id',
-            (args as FindDuplicatesResolverArgs).id,
-            fieldMetadataMap,
-          ),
-          data: await this.overrideDataByFieldMetadata(
-            (args as FindDuplicatesResolverArgs).data,
-            options,
-            fieldMetadataMap,
-            { shouldBackfillPosition: false },
+          ids: (await Promise.all(
```

Comment:
Do we expect a lot of ids? Should we run in batches instead?

Reply:
Just realised the operation bellow is not async and should not take much time though